Gems/Multiplayer/Code/Include/Multiplayer/MultiplayerTypes.h

```diff
@@ -88,6 +94,12 @@ namespace Multiplayer
         Activate
     };
 
+    enum class EntityMigration
+    {
+        Disabled,
+        Enabled
+    };
```

Comment:
Feels like this could just be a bool. Does this have potential to expand beyond 2 value types?

Reply:
No potential to expand, just better practice in APIs to use named literals rather than simple true/false values..

`myComponent.ConfigureNetParams(ReplicateToClients::Enabled, ReplicateToServers::Disabled, EntityMigration::Enabled);`

Reply:
The explicit param one is type-safe too to boot.. adds an extra layer of compile-time validation to code that touches API's that may change in the future.